pagefaultgames/pokerogue
# Various Error Fixes

## What are the changes?

There are no user-facing changes

## Why am I doing these changes?



This fixes all errors in non-node_modules files.

## What did change?


Types that were not explicitly defined have been defined. An improper function call was removed. A missing variable was declared.

## How to test the changes?




Encountering a Gym Leader or Elite 4 member in Classic mode is a quick way to re-verify the title localization code. Otherwise, really just a code check. These changes are too small and in too many files to reliably test. 

## Checklist
